iree/test/e2e/xla_ops/BUILD

```diff
@@ -164,79 +164,6 @@ iree_check_single_backend_test_suite(
             "round.mlir",
         ],
     ),
-    compiler_flags = [
-        "-iree-input-type=mhlo",
-        "-iree-llvm-link-embedded=true",
-    ],
-    driver = "dylib",
-    target_backend = "dylib-llvm-aot",
-)
-
-iree_check_single_backend_test_suite(
```

Comment:
Do we want to remove this? I would rather have a test suite with the `iree-llvm-link-embedded=false`. AFAIK we want to have the `false` path for triaging/debugging purposes.

Reply:
I was wondering about that too:

> Note: this switches test coverage from system linking to embedded linking. We may want to add back some test coverage for the system linker on each platform to ensure that debugging path remains functional.

I don't think iree/test/e2e/xla_ops/ (op specific integration tests) is the right place for such coverage though.

Reply:
How's this? I added back a smoketest in the compiler unit tests. I think the system linker's coverage should be a superset of the embedded linker that uses librt, so this should still be sufficient test coverage? We won't really have coverage for peculiarities around system linker flags (`-nostdlib`, `-shared`, etc.), but that might be okay?

Reply:
Ill let you make the call on this. I am fine with your approach.